### ISSUE TITLE
Axe Testing - Bug Fixes

### ISSUE THREAD

**Issue by AdiMakkar** [gctools-outilsgc/gccollab-frontend#151]
Footer testing failing. 
Furthermore, insights on failing tests for accessibility.

**Comment by AdiMakkar** [comment]
The following is the first issue: 

{
  "results": [
    {
      "path": "/features/footer/privacy.feature",
      "ok": false,
      "stepResults": [
        {
          "ok": false,
          "in": "be on the privacy webpage",
          "sourcePath": "/features/footer/privacy.feature",
          "actionResults": [
            {
              "ok": false,
              "message": "expected http://localhost:4200/privacy but on http://localhost:4200/home",
              "name": "beOnPage",
              "start": 3.995383966,
              "end": 3.998378551
            }
          ],
          "seq": 30
        }
      ]
    }
  ]
}


Usually the axe report displays accessibility issues but half of the time is returning the above.

**Comment by AdiMakkar** [comment]
Axe-html-reporter being added for tests for better readability and tracking.
